### PR TITLE
Appoint ACI as emission manager for LM programs - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Set ACI as Emission Manager"
```
Comment (sakulstra) [inline src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md:2]: Perhaps be more specific

Comment (marczeller) [reply src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md]: updated.

File: src/20240620_Multi_SetACIAsEmissionManager/AaveV3Arbitrum_SetACIAsEmissionManager_20240620.sol [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Arbitrum, AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IEmissionManager} from 'aave-v3-periphery/contracts/rewards/interfaces/IEmissionManager.sol';
+
+/**
+ * @title Set ACI as Emission Manager
+ * @author ACI
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x364de11d3a298f2c76721a8926cb32823cc29d0a95eadecbc0a98c628a38194b
+ * - Discussion: https://governance.aave.com/t/arfc-set-aci-as-emission-manager-for-liquidity-mining-programs/17898#arfc-set-aci-as-emission-manager-for-liquidity-mining-programs-1"
```
Comment (sendra) [inline src/20240620_Multi_SetACIAsEmissionManager/AaveV3Arbitrum_SetACIAsEmissionManager_20240620.sol:12]:
```suggestion
 * - Discussion: https://governance.aave.com/t/arfc-set-aci-as-emission-manager-for-liquidity-mining-programs/17898#arfc-set-aci-as-emission-manager-for-liquidity-mining-programs-1
```

File: src/20240620_Multi_SetACIAsEmissionManager/AaveV3Ethereum_SetACIAsEmissionManager_20240620.sol [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IEmissionManager} from 'aave-v3-periphery/contracts/rewards/interfaces/IEmissionManager.sol';
+
+/**
+ * @title Set ACI as Emission Manager
+ * @author ACI
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x364de11d3a298f2c76721a8926cb32823cc29d0a95eadecbc0a98c628a38194b
+ * - Discussion: https://governance.aave.com/t/arfc-set-aci-as-emission-manager-for-liquidity-mining-programs/17898#arfc-set-aci-as-emission-manager-for-liquidity-mining-programs-1"
```
Comment (sendra) [inline src/20240620_Multi_SetACIAsEmissionManager/AaveV3Ethereum_SetACIAsEmissionManager_20240620.sol:12]:
```suggestion
 * - Discussion: https://governance.aave.com/t/arfc-set-aci-as-emission-manager-for-liquidity-mining-programs/17898#arfc-set-aci-as-emission-manager-for-liquidity-mining-programs-1
```

File: src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Set ACI as Emission Manage for Aave Liquidity Mining programs"
```
Comment (sendra) [inline src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md:2]:
```suggestion
title: "Set ACI as Emission Manager for Aave Liquidity Mining programs"
```

File: src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Simple Summary
+
+The current ARFC proposes to appoint the Aave Chan Initiative (ACI) as the Emission Manager for liquidity mining programs across Aave instances. This will enable ACI to manage incentive programs funded by different partners, with the purpose of stimulating market activity and growth.
+
+## Motivation
+
+Liquidity mining programs are essential for attracting liquidity providers by offering rewards for their participation. Effective management of these programs is crucial for the sustained growth of the Aave Ecosystem. Therefore, ACI, with its extensive experience and strategic partnerships, is well-positioned to manage these emissions effectively.
+
+A recent example is the [Long Term Incentive Program (LTIPP) ](https://forum.arbitrum.foundation/t/aave-ltipp-application-final/21741) from Arbitrum DAO, where Aave DAO will receive 750,000 ARB tokens, illustrating the benefits of coordinated emission management.
```
Comment (sendra) [inline src/20240620_Multi_SetACIAsEmissionManager/SetACIAsEmissionManager.md:16]:
```suggestion
A recent example is the [Long Term Incentive Program (LTIPP)](https://forum.arbitrum.foundation/t/aave-ltipp-application-final/21741) from Arbitrum DAO, where Aave DAO will receive 750,000 ARB tokens, illustrating the benefits of coordinated emission management.
```

**Review by sendra — APPROVED** [review]

**Pull request merged**